test/bigi.js

```diff
@@ -196,6 +196,13 @@ describe('BigInteger', function () {
     }, /division by zero/)
   })
 
+  it('modInverse should always return a positive number', function () {
```

Comment:
Could also use a test that it should always return a number less than `m` (the big `fffff...` number). Inputs `88888888888888888888888888888888888888888888888888888888000cf80a` and `244846d2bdb089caa3ff959ec339ab85d4e32cb6a7770fa1c12c9f329f9a40c9` would fail that test without this fix.